test/main.cpp

```diff
@@ -0,0 +1,27 @@
+#include "ostream"
+
+#define CATCH_CONFIG_MAIN
+#include <catch.hpp>
+#include <mock_external.hpp>
+
+TEST_CASE("mock_external_c, set_send_reaction compiles", ""){
+    auto test_mock = r2d2::communication::mock_external_c();
+    auto frame = r2d2::frame_external_s();
+    std::function<void(
```

Comment:
The formatting makes this really hard to read. I know it is probably clang format that is causing this, but could you manually format this piece of code?

Reply:
I am aware, however I believe this is the most readable way this code can be written